src/examples/radiuss_tutorial/lesson_01/README.md

```diff
@@ -0,0 +1,285 @@
+# Lesson 01: Load an STL mesh
+
+In this lesson, we will begin developing our application by loading a triangle mesh in the STL format and printing out some of its properties.
+
+We use the [CLI11](https://github.com/CLIUtils/CLI11) library for parsing command line options. Axom vendors a copy of ``CLI11`` for internal use and provides it to downstream users. It is available via the `axom::cli11` CMake target.
+
+This example uses the following Axom components:
+* ``core`` -- has a dynamic array class ``axom::Array``
+* ``slic`` -- provides logging functionality
+* ``quest`` -- has a utility function for loading the STL mesh
+* ``mint`` -- provides a class for triangle meshes
+
+## Using a SLIC-based logger
+
+We begin by setting up a logger for our application using axom's `slic` component ([Simple Logging Interface Code](https://axom.readthedocs.io/en/develop/axom/slic/docs/sphinx/index.html)).
+
+`slic` uses a C-style API over a singleton instance and also provides a few basic logging macros that applications can either use or extend/build upon. 
+
+For our application, we will use an RAII-based wrapper struct
+
+```cpp
+struct BasicLogger
+{
+  BasicLogger()
+  {
+    namespace slic = axom::slic;
+
+    // Initialize the SLIC logger
+    slic::initialize();
+    slic::setLoggingMsgLevel(slic::message::Debug);
+
+    // Customize logging levels and formatting
+    const std::string slicFormatStr = "[lesson_01: <LEVEL>] <MESSAGE> \n";
+
+    slic::addStreamToMsgLevel(new slic::GenericOutputStream(&std::cerr),
+                              slic::message::Error);
+    slic::addStreamToMsgLevel(
+      new slic::GenericOutputStream(&std::cerr, slicFormatStr),
+      slic::message::Warning);
+
+    auto* compactStream =
+      new slic::GenericOutputStream(&std::cout, slicFormatStr);
+    slic::addStreamToMsgLevel(compactStream, slic::message::Info);
+    slic::addStreamToMsgLevel(compactStream, slic::message::Debug);
+  }
+
+  ~BasicLogger() { axom::slic::finalize(); }
+};
+```
+
+The constructor calls ``slic::initialize()``, sets the logging level to ``slic::message::Debug`` and then creates log streams for the different message levels. The destructor finalizes the slic component with `slic::finalize()`.
```

Comment:
```suggestion
The constructor calls ``slic::initialize()``, sets the logging level to ``slic::message::Debug`` and then creates log streams for the different message levels. The destructor finalizes the SLIC component with `slic::finalize()`.
```

Reply:
Is component the right word here in the last sentence?

Reply:
Reworded an capitalized.